tests/test_integration_1.py

```diff
@@ -156,6 +156,61 @@ def test_vertical_slider(self):
         for entry in self.get_log():
             raise Exception("browser error logged during test", entry)
 
+    def test_horizontal_slider_with_input(self):
```

Comment:
This looks like a great test, but let's not add any more tests to the top-level files `test_integration_1` or `test_integration_2`. These use the legacy `unittest` framework rather than `dash.testing`. If anything we should take opportunities when we're working in neighboring code to move more tests OUT of these files and port them to use `dash.testing` in the subdirectories.